autoload/clang_format.vim

```diff
@@ -223,14 +239,18 @@ function! clang_format#format(line1, line2) abort
     let source = join(getline(1, '$'), "\n")
     return s:system(clang_format, source)
 endfunction
+
+function! clang_format#format(line1, line2) abort
+    return clang_format#format_ranges([[line1, line2]])
+endfunction
 " }}}
 
 " replace buffer {{{
-function! clang_format#replace(line1, line2, ...) abort
+function! clang_format#replace(ranges, ...) abort
```

Comment:
Would you apply the same fix as `clang_format#format` here? Please avoid breaking change by adding new `clang_format#replace_ranges` and using it from `clang_format#replace`.